.github/workflows/auto-update.yml

```
@@ -0,0 +1,148 @@
+          echo "Regenerating SDK with latest OpenAPI specification..."
+          pnpm openapi:rebuild
+          pnpm build
+          echo "SDK regenerated successfully"
+      
+      - name: Test across Node versions
+        if: steps.check-changes.outputs.has_changes == 'true'
+        uses: ./.github/workflows/test-sdk.yml
+        with:
+          node-versions: '[18, 19, 20, 21, 22]'
+          upload-coverage: 'false'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)